documentatie/use-cases/validatieregels-plausibiliteitschecks-tellingen.md

```diff
@@ -31,375 +33,365 @@ Er zijn geen regels omdat het niet mogelijk is om foute aantallen in te vullen i
+> - Pas zo nodig rubriek 3.3.2 aan, en volg de instructies over hertellen die daar staan.
+
+Veld markeren: foutmelding op 3.3.1 "Vergelijk D en H"  
+
+#### DSO | F.351 "Vergelijk D&H": (checkbox D=H is aangevinkt, maar D<>H)
 
-Veld markeren: foutmelding op checkboxgroup Vergelijk D en H  
-Het standaard handelingsperspectief _voor de coördinator_ wordt bij deze foutmelding niet getoond (voor invoerder wel).
+> Invoerder: **Controleer je antwoorden** (F.351)
+
+> Coördinator: **De getallen die zijn ingevuld bij D en H zijn niet gelijk** (F.351)  
+> - Herstel de fout door op papier het juiste getal in te (laten) vullen.  
+> - Controleer ook of er een onverklaard verschil ontstaat.  
+> - Pas zo nodig rubriek 2.3.2 aan, en volg de instructies over hertellen die daar staan.
+
+Veld markeren: foutmelding op 2.3.1 "Vergelijk D en H"  
 
 #### CSO | F.302 "Vergelijk D&H": (checkbox H>D is aangevinkt, maar H<=D)
 
 > Invoerder: **Controleer je antwoorden** (F.302)
 
 > Coördinator: **Het getal dat is ingevuld bij H is niet groter dan D** (F.302)
 
-Veld markeren: foutmelding op checkboxgroup Vergelijk D en H  
+Veld markeren: foutmelding op 3.3.1 "Vergelijk D en H"  
 
 #### DSO | F.352 "Vergelijk D&H": (checkbox H>D is aangevinkt, maar H<=D)
 
 > Invoerder: **Controleer je antwoorden** (F.352)
 
 > Coördinator: **Het getal dat is ingevuld bij H is niet groter dan D** (F.352)  
-> Maak een corrigendum waarin de juiste optie geselecteerd wordt.
+> - Maak een corrigendum waarin de juiste optie geselecteerd wordt.
 
-Veld markeren: foutmelding op checkboxgroup Vergelijk D en H  
-Het standaard handelingsperspectief _voor de coördinator_ wordt bij deze foutmelding niet getoond (voor invoerder wel).
+Veld markeren: foutmelding op 2.3.1 "Vergelijk D en H"  
 
 #### CSO | F.303 "Vergelijk D&H": (checkbox H<D is aangevinkt, maar H>=D)
 
 > Invoerder: **Controleer je antwoorden** (F.303)
 
 > Coördinator: **Het getal dat is ingevuld bij H is niet kleiner dan D** (F.303)
 
-Veld markeren: foutmelding op checkboxgroup Vergelijk D en H  
+Veld markeren: foutmelding op 3.3.1 "Vergelijk D en H"  
 
 #### DSO | F.353 "Vergelijk D&H": (checkbox H<D is aangevinkt, maar H>=D)
 
 > Invoerder: **Controleer je antwoorden** (F.353)
 
 > Coördinator: **Het getal dat is ingevuld bij H is niet kleiner dan D** (F.353)  
-> Maak een corrigendum waarin de juiste optie geselecteerd wordt.
+> - Maak een corrigendum waarin de juiste optie geselecteerd wordt.
 
-Veld markeren: foutmelding op checkboxgroup Vergelijk D en H    
-Het standaard handelingsperspectief _voor de coördinator_ wordt bij deze foutmelding niet getoond (voor invoerder wel).
+Veld markeren: foutmelding op 2.3.1 "Vergelijk D en H"    
 
 #### CSO | F.304 "Vergelijk D&H": Meerdere aangevinkt of geen enkele aangevinkt
 
 > Invoerder: **Controleer je antwoorden** (F.304)
 
 > Coördinator: **Er moet precies 1 antwoord op deze vraag gegeven zijn** (F.304)
```

Comment:
Buiten scope van je changes, maar hier is denk ik: "Deze vraag moet precies één antwoord hebben." een net iets beter zin